server/service/src/invoice_line/stock_out_line/insert/generate.rs

```diff
@@ -95,8 +104,8 @@ fn generate_line(
         ..
     }: ItemRow,
     StockLineRow {
-        sell_price_per_pack,
-        cost_price_per_pack,
+        sell_price_per_pack: line_sell_price_per_pack,
```

Comment:
stock_line_sell_price_per_pack (sorry `line` to mean mean invoice_line)

Reply:
[4b7cdf9](https://github.com/msupply-foundation/open-msupply/pull/4756/commits/4b7cdf9042b08931d0a9f245037a6975c88e4826)